third_party/mkl/build_defs.bzl

```
@@ -81,18 +88,22 @@ def if_enable_mkl(if_true, if_false = []):
     })
 
 def mkl_deps():
-    """Shorthand for select() to pull in the correct set of MKL library deps.
-
-    Can pull in MKL-ML, MKL-DNN, both, or neither depending on config settings.
+    """Shorthand for select() to pull in the correct set of oneDNN library deps
+      depending on the plaform. x86 linux/windows with or without config=mkl will
```

Nit
```suggestion
      depending on the platform. x86 Linux/Windows with or without --config=mkl will
```
(Please also re-wrap the line.)

---

Yes, will fix with wrap.